vispy/gloo/wrappers.py

```diff
@@ -433,12 +433,18 @@ def get_state_presets(self):
         Returns
         -------
         presets : dict
-            The dictionary of presets usable with ``set_options``.
+            The dictionary of presets usable with :meth:`~set_state`.
```

Comment:
Did you try building the docs? Does this actually work? I would have assumed you needed `~vispy.gloo.wrappers.BaseGlooFunctions.set_state`

Reply:
For non-explicit names, Sphinx looks up in the current module. See [here](https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#python-roles) after the description of the roles.

But in this case '~' is not needed and it must be `:func:` instead of `:meth:` because the methods of `BaseGlooFunctions` are injected in and only visible in the global scope later on.

Reply:
And ```:func:`.set_state````referrs explicitly to this module's objects.